README.md

```diff
@@ -44,17 +46,17 @@ We appreciate your continued support, thank you!
 - .NET Standard 1.3 support
 - A SendGrid account, [sign up for free](https://sendgrid.com/free?source=sendgrid-csharp) to send up to 40,000 emails for the first 30 days or check out [our pricing](https://sendgrid.com/pricing?source=sendgrid-csharp).
 
-## Obtain API Key
+## Obtain an API Key
 
-You can obtain your API Key within the [SendGrid UI](https://app.sendgrid.com/settings/api_keys).
+Grab your API Key from the [SendGrid UI](https://app.sendgrid.com/settings/api_keys).
 
 ## Setup Environment Variables
 
-Update the development Environment with your [SENDGRID_API_KEY](https://app.sendgrid.com/settings/api_keys). For example, in Windows 10, please review [this thread](http://superuser.com/questions/949560/how-do-i-set-system-environment-variables-in-windows-10).
+Set your [SENDGRID_API_KEY](https://app.sendgrid.com/settings/api_keys) environment variable in your development environment. For example,[this thread](http://superuser.com/questions/949560/how-do-i-set-system-environment-variables-in-windows-10) explains how to set an environment variable in a Windows 10 system.
```

Comment:
I think this could be a little more clear still - when you set an environment variable, don't you have to name the variable? Or is it that you add your API key to your PATH? Either way, I think we could make it a bit more clear.

Reply:
How about: 

We advise against hard coding your [SENDGRID_API_KEY](https://app.sendgrid.com/settings/api_keys) in your application. Instead, please use an [environment variable](http://superuser.com/questions/949560/how-do-i-set-system-environment-variables-in-windows-10) or [Web.config](https://docs.microsoft.com/en-us/aspnet/identity/overview/features-api/best-practices-for-deploying-passwords-and-other-sensitive-data-to-aspnet-and-azure) or similar.